components/infobox/wikis/fighters/infobox_league_custom.lua

```diff
@@ -0,0 +1,196 @@
+	_args.icon = _args.icon or CustomLeague.getIconFromSeries(_args.series)
+
+	-- Normalize name
+	_args.game = Game.name{game = _args.game}
+
+	-- Swap prizepool to prizepoolusd when no currency
+	if not _args.localcurrency or _args.localcurrency:lower() == 'usd' then
+		_args.prizepoolusd = _args.prizepoolusd or _args.prizepool
+		_args.prizepool = nil
+	end
+
+	-- Implicit prizepools
+	_args.prizepoolassumed = false
+	if String.isEmpty(_args.prizepool) and String.isEmpty(_args.prizepoolusd) then
+		_args.prizepoolassumed = true
+
+		_args.singlesfee = tonumber(_args.singlesfee) or 0
+		_args.player_number = tonumber(_args.player_number) or 0
+		_args.singlesbonus = tonumber(_args.singlesbonus) or 0
+
+		local prizeMoney = _args.singlesfee * _args.player_number + _args.singlesbonus
+		if _args.localcurrency and _args.localcurrency:lower() ~= 'usd' then
+			_args.prizepool = prizeMoney
+		else
+			_args.prizepoolusd = prizeMoney
+		end
+	end
+
+	league.createWidgetInjector = CustomLeague.createWidgetInjector
+	league.defineCustomPageVariables = CustomLeague.defineCustomPageVariables
+	league.addToLpdb = CustomLeague.addToLpdb
+	league.getWikiCategories = CustomLeague.getWikiCategories
+
+	return league:createInfobox()
+end
+
+function CustomLeague:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomInjector:addCustomCells(widgets)
+	table.insert(widgets, Cell{name = 'Number of Players', content = {_args.player_number}})
+
+	return widgets
+end
+
+function CustomInjector:parse(id, widgets)
+	if id == 'customcontent' then
+		if _args.circuit or _args.points or _args.circuit_next or _args.circuit_previous then
+			table.insert(widgets, Title{name = 'Circuit Information'})
+			table.insert(widgets, Cell{name = 'Circuit', content = {_args.circuit}})
+			table.insert(widgets, Cell{name = 'Circuit Tier', content = {
+				_args.circuittier and (_args.circuittier .. ' Tier') or nil}
+			})
+			table.insert(widgets, Cell{name = 'Tournament Region', content = {_args.region}})
+			table.insert(widgets, Cell{name = 'Points', content = {_args.points}})
+			table.insert(widgets, Chronology{content = {next = _args.circuit_next, previous = _args.circuit_previous}})
+		end
+
+	elseif id == 'prizepool' then
+		if _args.prizepoolassumed then
+			widgets[1].content[1] = Abbreviation.make(
+				widgets[1].content[1],
+				'This prize is assumed, and has not been confirmed'
+			)
+		end
+
+	elseif id == 'gamesettings' then
+		return {
+			Cell{name = 'Game', content = {_args.game}},
+			Cell{name = 'Version', content = {_args.version}},
+		}
+	end
+	return widgets
+end
+
+function CustomLeague:addToLpdb(lpdbData, args)
+	lpdbData.participantsnumber = string.gsub(_args.player_number or '', ',', '')
+
+	if Logic.readBool(_args.overview) then
+		lpdbData.game = 'none'
+	end
+
+	lpdbData.extradata.assumedprizepool = tostring(_args.prizepoolassumed)
+	lpdbData.extradata.circuit = _args.circuit
+	lpdbData.extradata.circuit_tier = _args.circuit_tier
```

Comment:
should that also be stored into `lpdbData.publishertier`?

Reply:
Isn't currently, so no